### DISCUSSION TITLE
Reclaim disk space question

### DISCUSSION THREAD

**Discussion by kkarthik23** [concourse/concourse#6667] [Help & Support] [answered]
To reclaim space can we delete folders in /var/lib/concourse/volumes/live?

**Comment by taylorsilva** [comment]
Nope, if it's in that dir that mean's it's in use by Concourse. Deleting volumes in there will cause errors in your pipelines as you'll be deleting volumes that Concourse thinks exist and then suddenly don't!

Our recommendation is to increase work disks size in increments until you find your sweet spot for your workloads. 

For example, for everything we run on https://ci.concourse-ci.org/ we found that workers with 750GB disks is enough to handle all the volumes Concourse makes based on our pipelines. The disk size requirements will go up and down depending what your pipelines are doing.

**Reply by kkarthik23** [reply]
thanks @taylorsilva 
when we destroy the pipelines would baggage claim then delete these volumes which would then free up space?
just trying to understand how this works appreciate your reply.

**Comment by kkarthik23** [comment, marked as answer]
not exactly sure what happened but disk space has been recovered i did stop and start concourse worker after destroying pipelines and pruned worker  but i did that even before
